### PR TITLE
add FaST-LMM v0.6.3 to python3.sif 

### LINKED ISSUE #73: Consider adding pysnptools and fastlmm to python3.sif
[pysnptools](https://github.com/fastlmm/PySnpTools) seem to be handy tool for reading genotype data in different formates such as [plink's bed/bim/fam](https://www.cog-genomics.org/plink/1.9/formats#bed) format, or [bgen](https://www.well.ox.ac.uk/~gav/bgen_format/spec/latest.html).
I've already tried ``pysnptools`` in [popcorn](https://github.com/comorment/popcorn/blob/main/src/dockerfiles/popcorn/Dockerfile) container, it seem to work just fine.

Here is what's needed for the Dockerfile:
```
RUN mamba install \
    psutil=5.9.3 \
    more-itertools=9.0.0 \
    --yes

RUN yes | pip install cbgen==1.0.2
RUN yes | pip install bgen-reader==4.0.8 
RUN yes | pip install bed-reader==0.2.24
RUN yes | pip install pysnptools==0.5.7
```

### PR DESCRIPTION
**Pull request by ofrei**
Fixes #73 , by adding ``FaST-LMM`` (version 0.6.3) to future ``python3.sif`` build, and corresponding test

## Before submitting

- [x] I've read and followed all steps in the [Making a pull request](https://github.com/comorment/containers/blob/main/CONTRIBUTING.md#making-a-pull-request)
    section of the `CONTRIBUTING` docs.
- [x] If this PR adds a new feature, I've added tests that sufficiently cover my new functionality.

### CODE REVIEW

**Comment by ofrei** [comment]
This also reorders the definitions in python3 dockerfile so that they are sorted alphabetically.

**Review by espenhgn — APPROVED** [review]
Looks good to go! But may conflict with https://github.com/comorment/containers/pull/72 so let's decide what we merge first.

File: src/dockerfiles/python3/Dockerfile [hunk 1]
```diff
@@ -17,26 +17,38 @@ RUN bash install_mambaforge.sh
...
-    statsmodels=0.13.2 \
-    matplotlib=3.6.0 \
-    seaborn=0.12.0 \
-    pyyaml=6.0 \
     jupyterlab=3.4.8 \
+    --yes
+
+# keep the list of packages sorted alphabetically
+# https://www.online-utility.org/text/sort.jsp
+RUN mamba install \
```
[the remark this answers is not in the document]
Comment (ofrei) [reply src/dockerfiles/python3/Dockerfile:25]: This makes sense to keep as two separate RUNs.

**Comment by ofrei** [comment]
@espenhgn is it fine to do this https://github.com/comorment/containers/commit/e0e827f177f10f155f572b0c40e27c253532178f ?

Consolidating double RUN is a controversial suggestion, sometimes it makes sense to separate into separate runs
https://github.com/hadolint/hadolint/issues/715

**Comment by espenhgn** [comment]
> @espenhgn is it fine to do this [e0e827f](https://github.com/comorment/containers/commit/e0e827f177f10f155f572b0c40e27c253532178f) ?
> 
> Consolidating double RUN is a controversial suggestion, sometimes it makes sense to separate into separate runs [hadolint/hadolint#715](https://github.com/hadolint/hadolint/issues/715)

I can live with that. Perhaps for the records, explain in this thread why didn't want to have single `RUN` statements per mamba/pip install before merging (it's not quite obvious to me).

**Comment by ofrei** [comment]
A (vague) idea here was that it takes a fairly long time to build the "baseline" python package, and jupyterlab pulls many dependencies taking about half of the build time. It principle it would make sense to include other "heavy" packages in the first mamba install, while the second mamba install make be for smaller packages that are quick to install (thus it might save a bit of time once we rebuild after adding a small python package). But it's hard to draw a line between which packages to put in the first group vs which ones should go to the second. Also rebuilding them all from scratch is more robust - if there are issues we pick them up earlier. All in all, I think it's indeed best with one mamba install!

**Comment by espenhgn** [comment]
Yes, I understand. With consecutive `pip install ...` it'd be ok, but with `conda/mamba install ...` there is sometimes the case that earlier installs are updated/downgraded. In the latter case, I think it makes sense to have a single `RUN ...` statement. 
Thanks for merging!

**Pull request merged**